lib/inflector.php

```diff
@@ -238,7 +238,7 @@ public function camelize($term, $downcase_first_letter = self::UPCASE_FIRST_LETT
 
 				$m = $matches[0];
 
-				return !empty($acronyms[$m]) ? $acronyms[$m] : capitalize($m);
+				return !empty($acronyms[$m]) ? $acronyms[$m] : ucfirst($m);
```

Comment:
Ah right, good spotting.  Should there be tests for this (all current tests pass)